### ISSUE TITLE
Wrong startTime and EndTime for repeated events with no enddate

### ISSUE THREAD

**Issue by ABDows** [DHTMLX/scheduler-helper-php#11]
I create the following event :

```
start_date:2019-12-19 03:00:00
end_date:9999-02-01 00:00:00
rec_type:day_1___#no
event_pid:0
event_length:18000
```

and called getData Like :

``` PHP
        $start_ = "2020-01-01 00:00:00";
        $end_ = "2020-01-02 23:00:00";
                $all_event = $helper->getData($start_, $end_);
```

the result was : 

``` PHP
array:2 [▼
  0 => array:12 [▼
    "event_id" => "135"
    "start_date" => "2020-01-01 00:00:00"
    "end_date" => "2020-01-01 05:00:00"
    "blocked" => "1"
    "event_type" => "p"
    "prestation_id" => null
    "agenda_id" => "95"
    "rec_type" => "day_1___#no"
    "event_pid" => "0"
    "event_length" => "18000"
    "start_time" => 1577836800
    "end_time" => 1577854800
  ]
  1 => array:12 [▼
    "event_id" => "135"
    "start_date" => "2020-01-02 00:00:00"
    "end_date" => "2020-01-02 05:00:00"
    "blocked" => "1"
    "event_type" => "p"
    "prestation_id" => null
    "agenda_id" => "95"
    "rec_type" => "day_1___#no"
    "event_pid" => "0"
    "event_length" => "18000"
    "start_time" => 1577923200
    "end_time" => 1577941200
  ]
]
```

As you can see in the attached image file, the repeated events start at 3AM end finish at 8AM but scheduler helper return wrong time from 00AM to 05AM.

![Bug StartTime & EndTime](https://cloud.githubusercontent.com/assets/8752689/10716476/78746032-7b32-11e5-9c2d-acb719ae9cf8.png)

I think the problem is related to Y2K38 bug because the same error appears with repeated events having enddate great than 2038-01-19 03:14:08 but why it appeared in "2020-01-02" ?

Plz help !

**Comment by MrSef217** [comment]
Hello ,
i'm facing pretty much the same issue , is there a fix for this problem ?
please help.

**Comment by EndOfSource** [comment]
Hey mates,

I had a problem like this some weeks ago.

Does 

```
  $helper->config["server_date"] = true;
```

or 

```
$helper->config["server_date"] = false;
```

make a difference for you? For me the difference was always two hours. It had something to do with this setting and my timezone.

@MrSef217 @ABDows

**Comment by AlexKlimenkov** [comment]
If it's still actual, try the version from this commit  https://github.com/DHTMLX/scheduler-helper-php/commit/30987b06815c57f7630d368443ee4f92c94e2852
does it fix the issue?

**Comment by AlexKlimenkov** [comment]
Fixed in v1.1.0
https://github.com/DHTMLX/scheduler-helper-php/releases/tag/1.1.0
